Makefile

```
@@ -59,6 +59,9 @@ test_ci:
 	echo "Detected $(NPROCS) CPUs running with $(CONCURRENCY) workers"
 	stestr run --concurrency $(CONCURRENCY)
 
+test_visualizations:
+	 QISKIT_TESTS='force_visualization' python -m unittest discover -v test.python.visualization
```

We should run this in parallel with stestr to be faster and consistent with the other ci jobs:

```suggestion
	 QISKIT_TESTS='force_visualization' stestr run --concurrency $(CONCURRENCY) test.python.visualization
```